source/cBaseObject.cpp

```diff
@@ -2091,6 +2124,7 @@ bool CBaseObject::HandleLine( std::string &UTag, std::string &data )
 					SetHitChance( static_cast<SI16>( std::stoul( oldstrutil::trim( oldstrutil::removeTrailing( csecs[0], "//" )), nullptr, 0 )));
 					SetDefenseChance( static_cast<SI16>( std::stoul( oldstrutil::trim( oldstrutil::removeTrailing( csecs[1], "//" )), nullptr, 0 )));
 					SetSwingSpeedIncrease( static_cast<SI16>( std::stoul( oldstrutil::trim( oldstrutil::removeTrailing( csecs[2], "//" )), nullptr, 0 )));
+					SetDamageIncrease( static_cast<SI16>( std::stoul( oldstrutil::trim( oldstrutil::removeTrailing( csecs[3], "//" )), nullptr, 0 )));
```

Comment:
This might trigger a crash if trying to load a set of worldfiles that does not have the damageIncrease data in the EXTPROPCOMMON worldfile tag already. Enclose in if( csecs.size() >= 4 ):
```suggestion
					if( csecs.size() >= 4 )
					{
						SetDamageIncrease( static_cast<SI16>( std::stoul( oldstrutil::trim( oldstrutil::removeTrailing( csecs[3], "//" )), nullptr, 0 )));
					}
```